.thumbs.yml

```diff
@@ -1,6 +1,6 @@
 minimum_reviewers: 2
 build_steps:
- - make clean
```

Comment:
@javajolt re: clean state, should be a freshly cloned build dir for each head_sha/base_sha combo.